plugin/helplink.vim

```diff
@@ -14,9 +14,9 @@ let s:save_cpo = &cpo
 set cpo&vim
 
 let g:helplink_formats = {
-\	'markdown':  "'[`:help ' . l:tagname . '`](' . l:url . ')'",
-\	'html': "'<a href=\"' . l:url . '\"><code>:help ' . l:tagname . '</code></a>'",
-\	'bbcode': "'[url=' . l:url . '][code]:help ' . l:tagname . '[/code][/url]'"
+\	'markdown':  "'[`%s' . l:tagname . '`](' . l:url . ')'",
```

Comment:
Wouldn't it be easier to remove the `:help` here if you don't want it? Or add a new `markdown_nohelp` without the `:help`?

Reply:
I thought about the markdown_nohelp, but then one would need another one for `:h` so I thought the `%s` part is the most flexible.

Reply:
Do you want to have formats for `:help`, `:h`, and no tag at all, all at the same time?

Reply:
Not usually. But once I need it, I don't want to rewrite the plugin, so I prefer to give whatever I need as prefix to the htmllink#link() function.

Reply:
I don't know; adding two variable substitution mechanisms (`eval()` and `printf()` seems a bit meh to me. I pushed a commit to your PR to add additional predefined templates, and you should be able to do what you want with:

    nnoremap <silent> \h :let @h=helplink#link('markdown_nt')<cr>

What do you think? Does t his work for you?

Reply:
that works okay. However personally I don't like the duplication of so many templates. But I guess in practice it doesn't matter